src/sentry/static/sentry/app/views/userFeedback/organizationUserFeedback.tsx

```diff
@@ -39,10 +53,20 @@ class OrganizationUserFeedback extends AsyncView {
     ];
   }
 
-  getTitle() {
+  getTitle(): string {
```

Comment:
I think it can just infer this actually, especially if the `AsyncView` sets the type